cmd/export_ledger_entry_changes.go

```diff
@@ -71,14 +73,29 @@ be exported.`,
 			endNum = math.MaxInt32
 		}
 
-		ledgerCloseMeta, err := env.GetLedgerCloseMeta(endNum)
+		ledgerRange := ledgerbackend.BoundedRange(startNum, endNum)
```

Comment:
I think we want to skip adding ledger_closed_at to any of these ledger entry changes. The ledgerCloseMeta may not be of the correct ledger sequence because the changes for a batch all go though a change_compactor that may delete/create/compact certain changes together meaning their seq # could be wrong which means the ledger_closed_at time could be incorrect.

cc: @sydneynotthecity

Reply:
What do you think is the best way to skip the insertion of ledger_closed_at in this case?

Reply:
We probably want to update the change compactor in the go monorepo so that it will include the ledger seq of the compacted changes. TBD on those changes though